src/main/java/uk/gov/ons/ctp/response/collection/exercise/service/EventService.java

```diff
@@ -337,6 +338,21 @@ public boolean isScheduled(UUID collexUuid) throws CTPException {
         >= numberOfMandatoryEvents;
   }
 
+  @Transactional
+  public void updateEventStatus(CaseActionEventStatusDTO eventStatus) {
+    Event eventToBeUpdated =
+        eventRepository.findOneByCollectionExerciseIdAndTag(
+            eventStatus.getCollectionExerciseID(), eventStatus.getTag().toString());
+    if (null == eventToBeUpdated) {
```

Comment:
could this be an empty value instead of null

Reply:
can it be?